File: app/Http/controller/test_folders/app/Http/Controllers/UserController.php
use App\Http\Controllers\Controller;
use App\Models\Team;
use App\Models\User;
use Illuminate\Http\Request;

class UserController extends Controller
{
    public function index()
    {
        $users = User::latest()->get();

        return view('users', compact('users'));
    }

    public function show(User $user)
    {
        return view('users.show', compact('user'));
    }

    public function update(Request $request, User $user)
    {
        $validatedData = $request->validate([
            'team_id' => 'required',
            'profile' => 'required|string',
            'tags' => 'required|array',
        ]);

        $user->update($validatedData);

        return redirect()->route('users.index');
    }
}

File: contbk/test_folders/app/controllers/user_controller.py
from app.models.user import User

class UserController:
    def __init__(self):
        self.users = []

    def create_user(self, name: str, email: str) -> User:
        user = User(len(self.users) + 1, name, email)
        self.users.append(user)
        return user

    def get_users(self) -> list[User]:
        return self.users